test/test-dbPatients.js

```diff
@@ -36,15 +36,15 @@ describe('DB Patients', () => {
 
   describe('insertPatientInPG', () => {
     it('should INsert one patient in PG', async () => {
-      const psychologistId = '357a2085-6d32-44db-8fe6-979c7339fd47';
+      const psy = await clean.insertOnePsy();
```

Comment:
Since we now insert psy, shouldn't we clean all psychologists in `afterEach` or preferably `after`?

Reply:
not really, best practice is that when you want to have a test relying on existing data you clean it before, not expecting the test adding stuff to clean after itself

Reply:
oh ok, I wasn't doing it like this, I would now ;)